index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+   
+
+    <link rel="stylesheet" href="style.css">
+    <title>Layal's Personal Website</title>
+</head>
+<body>
+        <div class="nav-home"></div>
+    
+                <!-- (div>a{option$})*3 -->
+                
+              </div>
+              
+              <h1>Layal's Portfolio</h1>
+
+
+     <!-- Header start -->
+        <div class="header">
+                <div class="header-right">
+                  <a class="active" href="#home">Home</a>
+                  <a href="file:///Users/layalkashgari/code/wdi/hoemwork/hw-w02d01-html-and-css/portfolio.html">Portfolio</a>
```

Comment:
this kind of href only works on your computer you need to write a relative path not an absolute